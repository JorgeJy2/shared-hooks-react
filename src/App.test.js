import { shallow } from "enzyme";
import App from "./App";

describe('app test',()=>{

  test('should wrapper', () => {
      const wrapper = shallow(<App/>);
      expect(wrapper).toMatchSnapshot();
  });
  
});
