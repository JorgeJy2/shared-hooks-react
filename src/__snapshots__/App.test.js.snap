// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app test should wrapper 1`] = `
<div
  className="App"
/>
`;
